fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -173,15 +173,21 @@ def self.load_dot_env(env)
       return unless base_path
       require 'dotenv'
 
-      Actions.lane_context[Actions::SharedValues::ENVIRONMENT] = env if env
-
       # Making sure the default '.env' and '.env.default' get loaded
       env_file = File.join(base_path, '.env')
       env_default_file = File.join(base_path, '.env.default')
       Dotenv.load(env_file, env_default_file)
 
-      # Loads .env file for the environment passed in through options
-      if env
+      return unless envs
+
+      envs = envs.split(",") # multiple envs?
```

Comment:
Maybe call the arg "env_descriptor" or similar? Overwriting the var makes this harder to scan read